README.md

```diff
@@ -733,11 +770,11 @@ this way of variable usage is fine:
 ```php
 <?php
 
-function canModify($object)
+function canModify(ValueClass $object): bool
 {
     $rightsGranted = isAdmin() || isOwner($object);
     $objectEditable = isNew($object) && !isLocked($object);
```

Comment:
Looks like antipattern too, it's more efficient to apply https://refactoring.guru/replace-nested-conditional-with-guard-clauses in this case, especially if checks are not straightforward / take some time.